.kokoro/test-samples.sh

```diff
@@ -87,11 +87,11 @@ for file in samples/**/requirements.txt; do
     python3.6 -m nox -s "$RUN_TESTS_SESSION"
     EXIT=$?
 
-    # If this is a periodic build, send the test log to the FlakyBot.
-    # See https://github.com/googleapis/repo-automation-bots/tree/master/packages/flakybot.
+    # If this is a periodic build, send the test log to the Build Cop Bot.
```

Comment:
Out-of-date. Maybe needs https://github.com/googleapis/python-datacatalog/pull/120 first